cratonclient/shell/v1.py

```diff
@@ -0,0 +1,20 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+#TODO (cmspence): from cratonclient.v1 import regions_shell
```

Comment:
Why would we import those? We don't need individual shell modules for each resource there. We should be interacting only with `cratonclient.v1.client.Client` in my opinon.

We create one client and use that for the command the user passes.